### PR TITLE
Added a Select component

### PR DESCRIPTION
**Pull request by sarathms**
Custom styling on the native `<select>` element.

### CODE REVIEW

File: components/atoms/Select.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  padding: 10px 40px 10px 15px;
+  border-radius: 2em;
+  cursor: pointer;
+  background-color: ${props => props.theme.colors.gray1};
+  border: none;
+  outline: none;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAh0lEQVQ4T93TMQrCUAzG8V9x8QziiYSuXdzFC7h4AcELOPQAdXYovZCHEATlgQV5GFTe1ozJlz/kS1IpjKqw3wQBVyy++JI0y1GTe7DCBbMAckeNIQKk/BanALBB+16LtnDELoMcsM/BESDlz2heDR3WePwKSLo5eoxz3z6NNcFD+vu3ij14Aqz/DxGbKB7CAAAAAElFTkSuQmCC');
```
Comment (hellais) [inline components/atoms/Select.js]: I am not sure what this image is, but we should probably be either injecting the data-uri using some babel plugin or reference an image path and defer the decision of using a data-uri or something else to the webpack of whatever is using it.

Comment (hellais) [reply components/atoms/Select.js]: Another thing to keep in mind when using rasters is that it's usually best to have versions of the raster at 2x or 3x the resolution to support high density displays. For that reason it's actually probably best to have a vector image be the source and then using some tool to convert it to a raster.

**Comment by hellais** [comment]
I would also suggest you merge master into this branch (or rebase it into master) so that we can have the storybook preview as part of it.

**Comment by sarathms** [comment]
The first attempt I made was to use the `:after` selector with `{ content: '▾'}` in it. But turns out `<select>` doesn't honor it. Will try a pure CSS solution.

**Comment by sarathms** [comment]
I have now added a CSS based solution based on [this](https://css-tricks.com/snippets/css/css-triangle/). If we need this triangle pointing in different directions in more places, we can refactor it into its own file. (Then, since it is a constituent for an `atom`, would this warrant a category of its own like `components/electrons`? :stuck_out_tongue:

**Comment by hellais** [comment]
Yes this looks great. Good work!

**Pull request merged**